### PR TITLE
adobe-team feedback carousel changes

### PR DESCRIPTION
**Pull request by amlan18041996**
Fix #561

Test URLs:
- Before: https://main--danaher-ls-aem--hlxsites.hlx.page/
- After: https://503-implement-carousel-ui-block--danaher-ls-aem--hlxsites.hlx.page/us/en/products
- After: https://503-implement-carousel-ui-block--danaher-ls-aem--hlxsites.hlx.page/us/en/product1
- After: https://503-implement-carousel-ui-block--danaher-ls-aem--hlxsites.hlx.page/us/en/product3

### CODE REVIEW

**Comment by davenichols-DHLS** [comment]
@amlan18041996  - just quickly, on the H2 title on the carousel can we add     margin-top: 2px;  so that it is closer to the Brand Name?

image

File: styles/tailwind.css [hunk 1]
```diff
@@ -92,7 +92,7 @@
     @apply h-full w-full;
   }
 
-  main .section {
+  main .section:not(.carousel-container) {
```
Comment (duynguyen) [inline styles/tailwind.css:95]: don't we have the `main .section.carousel-container` styles already right below? let's keep either this or that one, not both.

File: scripts/Carousel.js
Comment (mhaack) [inline scripts/Carousel.js:1]: I know this was there before. But given that logic is part of the carousel block I would keep it there. No need for extra file.

**Review by davenichols-DHLS — APPROVED** [review]
This is excellent, thank you - please merge

**Pull request merged**